prog/dftb+/lib_dftbplus/mainio.F90

```diff
@@ -2899,12 +2957,17 @@ subroutine writeDetailedOut1(fd, iDistribFn, nGeoSteps, iGeoStep, tMD, tDerivs,
           & energy%EForceRelated * Hartree__eV, 'eV'
     end if
     if (tPeriodic .and. pressure /= 0.0_dp) then
-      write(fd, format2U) 'Gibbs free energy', energy%Etotal - sum(TS) + cellVol * pressure,&
-          & 'H', Hartree__eV * (energy%Etotal - sum(TS) + cellVol * pressure), 'eV'
+      write(fd, format2U) 'Gibbs free energy', energy%EGibbs, 'H', Hartree__eV * energy%EGibbs, 'eV'
```

Comment:
Is this change related to MBD?

Reply:
No, it wasn't, probably a merge left-over.